src/components/input/Demo.vue

```diff
@@ -200,6 +221,8 @@ export default defineComponent({
         const suffixProp = ref('100');
         const suffixSlot = ref('Feather-weight');
         const placeholder = ref<number|null>(null);
+        const helpProp = ref('I\'m an input!');
+        const helpSlot = ref('I\'m an input!');
```

Comment:
One is enough here

Reply:
Altough I realise no point of having v-models at all given the component isn't updated from the outside